lib/tasks/federal_email_domains.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'faraday'
+require 'csv'
+
+DOT_GOV_DOWNLOAD_PATH = 'https://raw.githubusercontent.com/cisagov/dotgov-data/main/current-federal.csv'
+namespace :federal_email_domains do
+  task load_to_db: :environment do |_task, _args|
+    response = Faraday.get(DOT_GOV_DOWNLOAD_PATH)
+
+    csv = CSV.parse(response.body, col_sep: ',', headers: true)
+    csv.each do |row|
+      FederalEmailDomain.find_or_create_by(name: row['Domain name'])
+    end
```

Comment:
This is _probably_ fine since we're only dealing with about 1400 entries, but I'd wonder if this should work a little more like the disposable emails task using `insert_all`. I think `upsert_all` could be an option too if we wanted to gracefully handle duplicates. I think the "find" element here is kinda wasted since we don't care to do anything with the result, so we could also just insert and rescue and discard on a `ActiveRecord::RecordNotUnique` exception.

If this was a bigger file, I'd also suggest using [`CSV.foreach`](https://ruby-doc.org/stdlib-2.6.1/libdoc/csv/rdoc/CSV.html#method-c-foreach) instead of `CSV.parse` + `Array#each`, since it'd process row-by-row instead of loading the whole thing into memory first.